check.py

```diff
@@ -175,6 +175,31 @@ def do_asm2wasm_test():
               fail_with_error('wasm interpreter error: ' + err) # failed to pretty-print
             fail_with_error('wasm interpreter error')
 
+        # verify debug info
+        if 'debugInfo' in asm:
+          jsmap = 'a.wasm.map'
+          cmd += ['--binarymap-file', jsmap,
```

Comment:
That is correct. If --source-map is specified, asm2wasm and wasm-as emit additional JSON source map file. If --source-map-url is specified, the optional custom section section "sourceMappingURL" will be added to the wasm file.

If --source-map is specified for wasm-dis, this JSON source map file is used to read location information and add these to the wast output.